packages/@azure/servicebus/data-plane/test/testUtils.ts

```diff
@@ -190,3 +191,21 @@ export function getReceiverClient(
 
   throw new Error("Cannot create receiver client for give client type");
 }
+
+export async function purge(
+  recieverClient: QueueClient | SubscriptionClient,
+  useSessions?: boolean
+): Promise<void> {
+  const peekedMsgs = await recieverClient.peek();
+  if (!peekedMsgs.length) {
+    return;
+  }
+
+  const receiver = useSessions
+    ? await recieverClient.getSessionReceiver()
+    : recieverClient.getReceiver();
+
+  receiver.receive(() => Promise.resolve(), (err) => console.log(`Error when purging: ${err}`));
+  await delay(3000);
```

Comment:
Would update this to 5 seconds.
Do any of our tests send more than 2 messages? If so, I would use 10 seconds, or using receiveBatch() based on peeked length maybe more effective

Reply:
I dont want to rely on how many messages we think the tests send.
We might increase that in the future, and then this will break.
Also consider the scenario where when testing locally, we might be using queues that we used for other purposes.

Reply:
Increased delay